src/DistributedCache.AzureTableStorage.V3/DistributedCache.AzureTableStorage.V3.csproj

```diff
@@ -34,12 +34,12 @@
 		<PackageReference Include="JetBrains.Annotations" Version="2022.1.0">
 			<PrivateAssets>All</PrivateAssets>
 		</PackageReference>
-		<PackageReference Include="Microsoft.Extensions.Azure" Version="1.6.0" />
```

Comment:
To be better compatible with .NET 6, keep the versions for .NET 6 and add a new section for .NET 8

Reply:
Thanks for the update, however this is not what I had in mind.

You can just use this in the V3 csproj
```xml
<TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</TargetFrameworks>
```

And update dependencies with condition like:

``` xml
<ItemGroup>
		<PackageReference Include="JetBrains.Annotations" Version="2022.1.0">
			<PrivateAssets>All</PrivateAssets>
		</PackageReference>
		<PackageReference Include="Microsoft.Extensions.Azure" Version="1.6.0" />
		<PackageReference Include="Newtonsoft.Json.Bson" Version="1.0.2" />
		<PackageReference Include="Azure.Data.Tables" Version="12.x.x" />
		<PackageReference Include="Stef.Validation.Options" Version="0.1.0" />
		<PackageReference Include="System.Linq.Async" Version="6.0.1" />
</ItemGroup>

<ItemGroup Condition="'$(TargetFramework)' != 'net8.0">
		<PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="6.0.0" />
		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
		<PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
</ItemGroup>

<ItemGroup Condition="'$(TargetFramework)' == 'net8.0">
		<PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="8.0.0" />
		<PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
		<PackageReference Include="Microsoft.Extensions.Options" Version="8.0.0" />
</ItemGroup>
```

Reply:
👍 that's cleaner. I'll create a new PR rather than keeping reverting changes